src/PKSim.Core/Services/ParameterDefaultStateUpdater.cs

```diff
@@ -0,0 +1,35 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Utility.Extensions;
+
+namespace PKSim.Core.Services
+{
+   public interface IParameterDefaultStateUpdater
+   {
+      void UpdateDefaultFor(ISpatialStructure spatialStructure);
+      void UpdateDefaultFor(IEventGroupBuildingBlock eventGroupBuildingBlock);
+   }
+
+   public class ParameterDefaultStateUpdater : IParameterDefaultStateUpdater
+   {
+      public void UpdateDefaultFor(ISpatialStructure spatialStructure)
+      {
+         //TODO implement
+      }
+
```

Comment:
open TODO

Reply:
yes see other comment. This will be done in another PR
https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/543